src/kOS.Safe/Persistence/HarddiskFile.cs

```diff
@@ -1,22 +1,21 @@
-﻿using kOS.Safe.Persistence;
-
-namespace kOS.Safe.Encapsulation
+﻿namespace kOS.Safe.Persistence
 {
     [kOS.Safe.Utilities.KOSNomenclature("VolumeFile", KOSToCSharp = false)]
     public class HarddiskFile : VolumeFile
```

Comment:
Should we consider "Harddisk" as two words ("hard disk") or one compound word ("harddisk")?  If we consider it two words, we should capitalize the "D", otherwise not.